src/helpers/bar.js

```diff
@@ -1,3 +1,5 @@
+/* global cozy */
```

Comment:
NIT : I'm open to some explanations by Zoom for this commit

Reply:
Done